build/chip/tests.gni

```diff
@@ -12,13 +12,15 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import("//build_overrides/build.gni")
 import("//build_overrides/chip.gni")
 
+import("${build_root}/config/linux/cross_build.gni")
 import("${chip_root}/src/platform/device.gni")
 
 declare_args() {
   # Enable building tests.
-  chip_build_tests = current_os != "freertos"
+  chip_build_tests = current_os != "freertos" && !is_crossbuild
```

Comment:
What's the issue with building the tests? We should be able to build them although we won't be able to run them in general without wrapping with qemu.

Reply:
Moved this check to chip_link_tests below, and we can consider running the tests using QEMU later.